source/model/Device.py

```diff
@@ -6,16 +6,15 @@ class Device(ABC):
     An abstract implementation of a device
     Concrete plugins should implement his device.
     """
-    cntr = 0
+    cntrActivators = 0
 
     def __init__(self):
-        self.deviceId = self.__class__.newId()
         self._activators = list()
 
-    @classmethod
-    def newId(cls):
-        cls.cntr += 1
-        return cls.cntr
+    @property
```

Comment:
The same as with Activator.
It would be neither if the device implements the id him self.